src/lib.rs

```diff
@@ -308,42 +312,44 @@ impl<T, E> UnwrapCapOverflow<T> for Result<T, E> {
 #[cfg_attr(all(feature = "gecko-ffi", any(test, miri)), repr(align(8)))]
 #[repr(C)]
 struct Header {
-    _len: SizeType,
-    _cap: SizeType,
+    len: SizeType,
+    cap: SizeType,
```

Comment:
Given that this crate is designed to support FFI with an equivalent vectors type in Gecko's C++ code, I think we may need to keep the field names as they were. It's probably possible to change them here with an change to Gecko's bindgen code, but also probably not worth the bother?

Can we just ignore this lint?

Reply:
No need to do this, the names are already different in Gecko.